projects/ngx-skeleton-loader/src/lib/ngx-skeleton-loader.html

```diff
@@ -10,14 +10,12 @@
     [attr.aria-valuetext]="loadingText()"
     role="progressbar"
     tabindex="-1"
-    [ngClass]="{
```

Comment:
nit: Another approach could be just replacing from `[ngClass]` to `[class]` since `[class]` will also accept an object